src/main/java/net/dries007/tfc/util/ICollapsableBlock.java

```diff
@@ -1,121 +0,0 @@
-/*
```

Comment:
Sure. The API portion is `FallingBlockManager` now, blocks can do what they could do before and slightly more there instead of implementing via interfaces.